backend/src/test/java/gov/cdc/usds/simplereport/service/TestResultUploadServiceTest.java

```diff
@@ -385,8 +386,8 @@ void uploadService_getUploadSubmission_processingModeCode_doesntOverrideFileValu
     // THEN
     verify(dataHubMock).uploadCSV(fileContentCaptor.capture());
     String[] rows = new String(fileContentCaptor.getValue(), StandardCharsets.UTF_8).split("\n");
-    assertThat(rows[0]).endsWith(",processing_mode_code");
-    assertThat(rows[1]).endsWith(",D");
+    assertThat(rows[0]).endsWith(",\"processing_mode_code\"");
+    assertThat(rows[1]).endsWith(",\"D\"");
```

Comment:
How come this changed? Is this the effect of the `ALWAYS_QUOTE_STRINGS` feature for the `CsvMapper` I'm guessing?

Reply:
yes, that's a correct guess

Reply:
It was adding quotes to strings, but not always, and I added the `ALWAYS_QUOTE_STRINGS` setting to make it consistent